### ISSUE TITLE
Missing ANSI Escape sequence to reset color after output?

### ISSUE THREAD

**Issue by Hrxn** [epogrebnyak/justpath#12]
Hey, nice program, really like it!

One thing I noticed with justpath (Running on Windows 10 / Latest Windows Terminal / Latest PowerShell), not sure if this is platform specific:

![fin](https://github.com/epogrebnyak/justpath/assets/12064918/928005d1-650e-4fdd-86e1-3b6bb5881f4a)

The `PS C:\` which is displayed as the internal prompt function of PowerShell still has the same foreground color as the output generated by justpath..

**Comment by epogrebnyak** [comment]
Thanks for using `justpath` and reporting the problem with color. Let me find out how to paint the console back to original color in colorama

**Comment by epogrebnyak** [comment]
Will need to print(Style.RESET_ALL) to reset, will add shortly to code.

**Comment by epogrebnyak** [comment]
See if `pip install -U justpath==0.0.13` works as intended

**Comment by Hrxn** [comment]
![justpath](https://github.com/epogrebnyak/justpath/assets/12064918/5d13d937-dd99-4586-888c-bae4e03d7494)

So yeah, the ANSI SGR reset is there and works, but it also adds an additional newline because that's how `print` works in Python, I assume..

```py
if color:
        print(Style.RESET_ALL)   
```

Compare with `justpath --no-color`:

![jp2](https://github.com/epogrebnyak/justpath/assets/12064918/8693fabe-052c-4ed7-835a-d283de13875d)

I don't know, maybe simply append the ANSI reset sequence to the output of the last line or something?
Maybe there is a way in Python to `print()` without adding a newline, I'm not sure..



Oh, and by the way, I initially thought that this was an outlier due to how Windows (or rather PowerShell) works in not resetting ANSI options before printing the prompt, but then I discovered this:

https://asciinema.org/a/642726

Seems like it exhibits the exact same behavior there?

**Comment by Hrxn** [comment]
BTW, just checked this on my system here, to see what `colorama`'s `Style.RESET_ALL` does:

```py
>>>
>>>
>>>
>>> colorama.Style
<colorama.ansi.AnsiStyle object at 0x0000024148F81D60>
>>> colorama.Style.RESET_ALL
'\x1b[0m'
>>>
```

Yup, that is simply the ANSI SGR sequence for reset all/turn off all, `ESC[0m`

**Comment by epogrebnyak** [comment]
Thanks for testing this out, we need just print RESET_ALL without a new line, that I can mange, will post here soon.

**Comment by epogrebnyak** [comment]
@Hrxn - you seem to know a bit of each sequences, rare skill. That's a compliment

**Comment by epogrebnyak** [comment]
Should be fixed in `pip install -U justpath==0.0.14`

**Comment by Hrxn** [comment]
Yep, thank you!
Closing this now.
